tests/paren_space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -710,6 +715,13 @@ return (
   />
 );
 
+const SpaceOrNoSpace = styled.div\`
+  transform: \${( props ) =>
+    props.isComplete ? "rotateY(180deg)" : "rotateY(0)"};
+  transform: \${( props ) =>
+    props.isComplete ? "rotateY(180deg)" : "rotateY(0)"};
+\`;
+
```

Comment:
I would've expected paren spacing to add spaces in the `${}`. This seems to confirm the issue in #28.